youtube_dl/extractor/bbc.py

```diff
@@ -796,9 +795,12 @@ def _real_extract(self, url):
                             entries.append(self._extract_from_playlist_sxml(
                                 playlist.get('progressiveDownloadUrl'), playlist_id, timestamp))
 
+        playlist_title = self._html_search_regex(
```

Comment:
Place right here in the following order:
1. If no title extract with `_og_search_title`.
2. If still no title extract from `<title>`.
3. Extract `playlist_description`.